kimchi/src/prover.rs

```diff
@@ -165,13 +169,15 @@ where
     pub fn create_recursive<
```

Comment:
That seems to introduce a change also in the whole Mina repository. We might want to be more careful regarding this. Can we avoid this in this PR?

Reply:
We can't really, since then the prover is non-deterministic and it serializes to a different value every time.